packages/xod-arduino-deploy/src/boardsParser.js

```diff
@@ -42,19 +54,44 @@ const loadBoards = R.curry(
 //
 // =============================================================================
 
+// :: FQBN -> Map boardIdentifier BoardPrefs -> BoardPrefs
+const pickBoardPrefsByFqbn = R.curry(
+  (fqbn, boardPrefsMap) => {
+    const pabc = Utils.parseFQBN(fqbn);
+    const getCpuPrefs = R.path(['menu', 'cpu', pabc.cpu]);
```

Comment:
If you’d use `R.pathOr(['menu', 'cpu', pabc.cpu], {})` you can get rid of `R.when` with duplication below. This would make the code more linear and clear.